### DISCUSSION TITLE
What is the command to show the current page number?

### DISCUSSION THREAD

**Discussion by WandersFar** [sumatrapdfreader/sumatrapdf#3224] [Q&A] [answered]
[In the docs I saw this:](https://www.sumatrapdfreader.org/docs/Keyboard-shortcuts)

Shift + i - show current page number

I’d like to change that hotkey to something else, but “show current page number” or “Shift + i” aren’t listed [on the Commands page.](https://www.sumatrapdfreader.org/docs/Commands)

Does anyone know what the Command ID is? CmdPageNumber? Something else?

**Comment by WandersFar** [comment, marked as answer]
Never mind! Apparently it’s called something else in the Commands doc:

<html><body>


CmdTogglePageInfo | I | Toggle Page Info
-- | -- | --



</body>
</html>

Suggest that either the Keyboard Shortcuts or Commands page be edited so they use the same language.

**Comment by kjk** [comment]
I've updated the commands page to be similar to keyboard shortcuts page.
